yarn-project/end-to-end/src/e2e_nft.test.ts

```diff
@@ -61,23 +59,11 @@ describe('NFT', () => {
     const nftContractAsUser1 = await NFTContract.at(nftContractAddress, user1Wallet);
 
     // In a simple "shield" flow the sender and recipient are the same. In the "uniswap swap to private" flow
-    // it would be the uniswap contract.
+    // the sender would be the uniswap contract.
```

Comment:
Why not make these two different accounts? Also let's change 'uniswap' for 'amm'.

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/9150/commits/e938aca048ee9de9522f537816ae25c4580b5b38